spec/system/teacher_interface/authentication_spec.rb

```diff
@@ -245,8 +245,29 @@ def and_i_click_continue
   end
 
   def and_i_see_already_confirmed_message
-    expect(page).to have_content(
+    expect(sign_up_page).to have_content(
       "Your email address is already confirmed, please sign in."
     )
   end
+
+  def sign_in_page
+    @sign_in_page = PageObjects::TeacherInterface::SignIn.new
+  end
+
+  def sign_up_page
+    @sign_up_page = PageObjects::TeacherInterface::SignUp.new
+  end
+
+  def check_email_page
+    @check_email_page = PageObjects::TeacherInterface::CheckYourEmail.new
+  end
+
+  def new_application_form_page
+    @new_application_form_page =
+      PageObjects::TeacherInterface::NewTeacherForm.new
+  end
+
+  def signed_out_page
+    @signed_out_page = PageObjects::TeacherInterface::SignedOut.new
+  end
```

Comment:
These could be moved in to the `page_helpers.rb` file so they're accessible across all the tests (although I'd suggest prefixing them with `teacher_` since we will have assessor sign in pages too).